nwb_conversion_tools/datainterfaces/sipickledatainterfaces.py

```diff
@@ -21,51 +19,16 @@ def get_input_schema(cls):
         )
 
     def __init__(self, **input_args):
-        super().__init__(**input_args)
+        self.input_args = input_args
```

Comment:
why is this line needed?

Reply:
Previously, when this inherited from BaseDataInterface, calling super().__init__ calls self.input_args = input_args.

The other alternative here is to call super().super().__init__ but that seems a little convoluted (let me know if you'd prefer it, though).

Having the record of received input arguments is very useful for both debugging and retrieving information for stuff like metadata.

Reply:
why do we need input_args to be assigned as an attribute of the class?

Reply:
All inheritors of the [base data interfaces](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/nwb_conversion_tools/basedatainterface.py) have this. Downstream stuff may or may not wish to access it as a shortcut for retrieving file paths and stuff of that nature

Reply:
ok sure